laurat007/assignment2/AncestorsTest.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package assignment_2;
+
+import java.util.ArrayList;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import junit.framework.TestCase;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import static org.junit.Assert.*;
```

Comment:
Try to avoid importing the whole package. This can lead to some double imports sometimes. It is usually better to import only what you are using from a package :)

Reply:
It seems it was not even necessary that import.